perception/image_projection_based_fusion/src/roi_cluster_fusion/node.cpp

```diff
@@ -67,7 +69,8 @@ void RoiClusterFusionNode::postprocess(DetectedObjectsWithFeature & output_clust
   for (auto & feature_object : output_cluster_msg.feature_objects) {
     if (
       feature_object.object.classification.front().label !=
-      autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN) {
+        autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN ||
+      feature_object.object.existence_probability >= min_roi_existence_prob_) {
```

Comment:
[Q] This block is only called when `remove_unknown_ == true`. Is this as designed?
(I think it should work even if remove_unknown option is not true)